Registration.qmd

```diff
@@ -36,7 +36,7 @@ Upon registration, you will be prompted to verify your email address by receivin
 
 After successfully verifying your email, your registration process is complete. You can now log in using your credentials (providing Email and Password).
 
-If you have an issue with registering or you want to deregister, please [contact us](mailto://help-login@dataspace.copernicus.eu?Subject=Subject%20Text&Body=Your%20comments).
+If you have an issue with registering or you want to deregister, please [contact us](mailto://help-login@dataspace.copernicus.eu?Subject=Subject%20Text&Body=Your%20comments) directly.
```

Comment:
> ?Subject=Subject%20Text&Body=Your%20comments

It makes no sense to me to use these `subject` and `body` parameters if you are just putting generic placeholders anyway